docker-compose.yml

```diff
@@ -16,8 +16,22 @@ services:
     ports:
       - "5432:5432"
 
+  mongodb:
+    image: mongo:5.0.6
+    ports:
+      - "27017:27017"
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: opencraft
+      MONGO_INITDB_ROOT_PASSWORD: opencraft
+
+  mysqldb:
+    image: mysql:8.0.28-debian
```

Comment:
We're still using MySQL 5.6 on the production. Shouldn't we use the same version here?

Reply:
Updated to use 5.6